fiasco/__init__.py

```diff
@@ -4,6 +4,7 @@
 from fiasco.collections import IonCollection
 from fiasco.elements import Element
 from fiasco.fiasco import list_elements, list_ions, proton_electron_ratio
+from fiasco.gaunt import GauntFactor
```

Comment:
Are we sure we want this to live in the top level namespace? I imagine most user interaction with this to be via the `Ion` object.